db/schema.rb

```diff
@@ -147,8 +153,8 @@
     t.string "feedburner_orig_link"
     t.string "feedburner_orig_enclosure_link"
     t.boolean "is_perma_link"
-    t.datetime "source_updated_at", precision: nil
     t.string "keyword_xid"
+    t.datetime "source_updated_at", precision: nil
```

Comment:
Weird this field keeps flipping around.  I'm guessing the order of the fields is different in staging vs prod?